go/enclave/storage/enclavedb/batch.go

```diff
@@ -79,10 +79,10 @@ func WriteTransactions(ctx context.Context, dbtx *sql.Tx, transactions []*core.T
 
 			args = append(args, transaction.Tx.Hash()) // tx_hash
 			args = append(args, txBytes)               // content
-			args = append(args, toContracts[i])        // To
+			args = append(args, toContractIds[i])      // To
 			args = append(args, transaction.Tx.Type()) // Type
-			args = append(args, senders[i])            // sender_address
-			args = append(args, i)                     // idx
+			args = append(args, senderIds[i])          // sender_address
+			args = append(args, fromIdx+i)             // idx
```

Comment:
I take it this was wrong value previously

Reply:
I'm not sure about the offsets that were passed to the tx execution either